pytest_easyread.py

```diff
@@ -36,7 +36,10 @@ def __init__(self, reporter):
         self.is_first_failure = True
 
     def write_fspath_result(self, nodeid, res, **kwargs):
-        # inbuilt pytest reporter method; changes made: added **kwargs to add markup to path name
+        """
+        inbuilt pytest reporter method;
+        changes made: added **kwargs to enable adding markup to path name
```

Comment:
I'm not sure about having`changes made` in the docstring. If I'm a dev looking at this function for the first time, I want to know what the code does *now* - I don't need to know what was in the old version. My ideal docstring would contain: 

1) a line or two's summary of what the function does 
2) the type of each each parameter, with maybe an example if that param is particularly complex (you've done this correctly elsewhere!)
3) the type of the return value if there is one (and example if needed). If the function doesn't return anything e.g. but amends a global variable or some other state, mention that too.

So here I'd do something like 
```
"""
Extends the inbuilt pytest reporter method, which logs test output to console   # (at least I think that's what's happening here!)
:params nodeid: information about test path, class and name (type: string)
:params res: the result output (type: string)   
:params **kwargs: dict of options for adding markup to path name
:return: None
"""
```

Reply:
Thanks for this. I was not sure how to describe the functions I did not write, but that sounds good, I will apply it :). Thank you so much for taking time to review my code ^_^